fileglancer/tests/test_filestore.py

```diff
@@ -54,7 +54,7 @@ def test_get_root_info(filestore, test_dir):
     file_info = filestore.get_file_info(None)
     assert file_info is not None
     assert file_info.name == os.path.basename(test_dir)
-    assert file_info.path is None
+    assert file_info.path == '.'
```

Comment:
@allison-truhlar do you know if we ever expect this to be null? If we do I can revert the change for get_root_info